gridgencore.py

```diff
@@ -243,6 +244,10 @@ def validate_single_param(p: str, v):
 
 ######################### YAML Parsing and Processing #########################
 
+class GridYamlLoader(yaml.SafeLoader):
```

Comment:
Why the empty class?

Reply:
Oh i see, `pyyaml-include` shoves itself into classes and this empty class is needed to not pollute the base. Nice.